LocalSettings.php

```diff
@@ -3741,6 +3751,11 @@
 	// Rights
 	'+wgAvailableRights' => [
 		'default' => [],
+    		'+allpediawiki' => [
```

Comment:
Should be ordered alphabetically

Reply:
Also doesn't need the `+`

Reply:
> Should be ordered alphabetically

It is, `default` is always sorted above the optional wiki configurations. See prior revision.

Reply:
Below ahinfoboxeswiki

Reply:
Also doesn't need the `+` here.

Reply:
Done